js/src/components/modals/Modal.js

```diff
@@ -31,24 +44,45 @@ class Modal extends React.Component {
 		this.appElement = document.querySelector( this.props.appElement );
 	}
 
+	/**
+	 * Sets the Modal state to open.
+	 *
+	 * @returns {void}
+	 */
 	openModal() {
 		this.setState( {
 			modalIsOpen: true,
 		} );
 	}
 
+	/**
+	 * Sets the Modal state to closed.
+	 *
+	 * @returns {void}
+	 */
 	closeModal() {
 		this.setState( {
 			modalIsOpen: false,
 		} );
 	}
 
 	/**
-	 * Returns the rendered html.
+	 * Renders the Modal component.
 	 *
-	 * @returns {ReactElement} The rendered html.
+	 * @returns {ReactElement} The rendered react element.
 	 */
 	render() {
+		const defaultLabels = {
+			labels: {
```

Comment:
Having an inner `label` object seems unnecessary. Simply use the `defaultLabels` object.